src/datePicker/__snapshots__/DatePicker.unit.tsx.snap

```diff
@@ -40,7 +40,7 @@ exports[`DatePicker renderNavbar Should render the previous/next buttons in hori
   display: -webkit-inline-flex;
   display: -ms-inline-flexbox;
   display: inline-flex;
-  padding: 16px 24px;
+  padding: 0 24px;
```

Comment:
**[no magic numbers]**  Setting paddings according to brand spaces, eg: `space.x` to inforce typography grid

Reply:
This is in the snapshot, it's fine in index.tsx :)